Assets/HoloToolkit/Input/Scripts/InputSources/EditorInputSource.cs

```diff
@@ -49,7 +49,7 @@ public ButtonStates()
 
         public bool SupportsPosition;
         public bool SupportsRotation;
-        public bool SupportsPointing;
```

Comment:
SupportsPointing made sense since this promotes the point and commit interaction vs gaze and commit. Ray could be anything.
Thoughts?